graf3d/eve7/src/REveManager.cxx

```diff
@@ -1109,11 +1109,7 @@ void REveManager::GetServerStatus(REveServerStatus& st)
 {
    std::unique_lock<std::mutex> lock(fServerState.fMutex);
    gSystem->GetProcInfo(&fServerStatus.fProcInfo);
-#if defined(R__LINUX) || defined(_MSC_VER)
-   std::timespec_get(&fServerStatus.fTReport, TIME_UTC);
-#else
    fServerStatus.fTReport = std::time_t(nullptr);
```

Comment:
That probably doesn't work on Windows (let's see the PR build)

Reply:
@bellenot
Thx, I will check the result and add the _MSC_VER if there are errors.
Is Windows build triggered automatically?

Reply:
Sorry @alja , I compiled locally and pasted the error below..

Reply:
You could keep something like:
```
#if defined(_MSC_VER)
   std::timespec fTReport = {0, 0};
#else
   std::time_t   fTReport = 0;
#endif
```
and
```
#if defined(_MSC_VER)
   std::timespec_get(&fServerStatus.fTReport, TIME_UTC);
#else
   fServerStatus.fTReport = std::time_t(nullptr);
#endif
```

Reply:
@bellenot I have added the change now.